### PR TITLE
Check if migration jobs exists, and don't run consistency checker if …

### PR DESCRIPTION
**Pull request by bserdar**
…they do

### CODE REVIEW

File: migrator/src/main/java/com/redhat/lightblue/migrator/ConsistencyCheckerController.java [hunk 1]
```diff
@@ -131,67 +151,72 @@ public void run() {
...
-                    try {
-                        ae=lock(lockId);
-                    } catch (Exception e) {
-                        LOGGER.error("Exception during lock attempt {}:{}",lockId,e);
-                        ae=null;
-                    }
-                    if(ae!=null) {
-                        Breakpoint.checkpoint("CCC:locked");
+                    if(migrationJobsExist()) {
+                        LOGGER.error("There are migration jobs for {}, not running consistency checker this time",
```
Comment (paterczm) [inline migrator/src/main/java/com/redhat/lightblue/migrator/ConsistencyCheckerController.java]: Is this really an error?

Comment (bserdar) [inline migrator/src/main/java/com/redhat/lightblue/migrator/ConsistencyCheckerController.java]: Should I make it an info?

Comment (paterczm) [inline migrator/src/main/java/com/redhat/lightblue/migrator/ConsistencyCheckerController.java]: This is normal operation, right? Info should be fine.

Comment (bserdar) [inline migrator/src/main/java/com/redhat/lightblue/migrator/ConsistencyCheckerController.java]: It is info now.

**Pull request merged**